src/views/UserList/UserList.js

```diff
@@ -55,7 +56,7 @@ const UserList = ({ getUsersAction }) => {
     <div className={classes.root}>
       <UsersToolbar onSearchChange={onSearchChange} />
       <div className={classes.content}>
-        <UsersTable users={searchResults} />
+        {!searchResults.length ? <Spinner /> : <UsersTable users={searchResults} />}
```

Comment:
Can you try to add this spinner inside the table with users? As in the Latest Support Tickets:
![image](https://user-images.githubusercontent.com/44967506/82755951-43d84d80-9dd7-11ea-8022-a54cf9197b1c.png)

Reply:
Done